churchland_pipeline/setup.py

```diff
@@ -0,0 +1,5 @@
+import datajoint as dj
+
+dj.config['database.host'] = 'localhost'
```

Comment:
avoid placing hosts and credentials in public files. Instead, use local configuration files to be excluded from git.